src/components/LuxInputButton.vue

```diff
@@ -198,7 +198,6 @@ export default {
   &.icon-prepend {
     display: flex;
     align-items: center;
-    &:hover,
```

Comment:
On hover in light theme we would want
&:hover {
text-decoration-color: var(--color-princeton-orange-on-white);
}